styles/style.css

```diff
@@ -1,387 +1,305 @@
-  .cars-section h1 {
-    font-size: 4rem;
-  }
-  .image-container-cars {
-    display: flex;
-    flex-direction: row;
-    gap: 3rem;
-    margin: 5rem 1rem;
-  }
+.image-container-cars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  max-width: 100%;
+  padding: var(--padding-big) 0;
+}
 
-  .image-container-cars div {
-    width: 100%;
-  }
+.image-container-cars img {
+  padding: 0.5rem;
+  border: 2px solid white;
+  object-fit: contain;
+  height: 50vh;
+  width: auto;
 }
 
-/* Contact section */
+/*  ----------------Contact Section---------------- */
 .contact-section {
   background-image: url(/images/swansway-motor-group-wnz9awGVA9w-unsplash.jpg);
   background-size: cover;
   background-repeat: no-repeat;
+  background-position: center;
   text-align: center;
-  height: 100vh;
+  height: var(--dvh-with-fallback);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
-.heading-contact h1 {
-  letter-spacing: 0.2rem;
-  padding-left: 0.6rem;
-  padding-right: 0.6;
-  padding-top: 5rem;
-  margin-bottom: 2.6rem;
-  font-weight: 800;
+.contact-section h1 {
+  font-weight: var(--font-weight-heading);
+  font-size: var(--font-size-heading);
 }
 
-.paragraph-contact p {
-  font-size: 1.3rem;
-  margin-top: 0.8rem;
-  margin-bottom: 3rem;
-  padding: 0.5rem;
+.contact-section p {
+  font-weight: 500;
+  font-size: var(--font-size-paragraph);
 }
 
 .forms {
   display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
-  padding: 1rem;
-  margin: 1rem;
-  background-color: hsla(105, 6%, 13%, 0.466);
-  max-width: 1050px;
+  padding: var(--padding);
+  background-color: hsla(0, 0%, 0%, 0.616);
+  max-width: max-content;
+  margin-top: var(--padding);
+  gap: 1rem;
 }
 
 .input-group {
-  padding: 0.8rem;
+  display: flex;
+  flex-direction: column;
+  width: 35vw;
+  gap: 0.5rem
 }
 
-input,
-textarea {
-  width: 90%;
-  max-width: 1050px;
+.forms input,
+.forms textarea {
+  resize: none;
   padding: 0.5rem;
-  outline: 0;
   border: 1px solid rgb(240, 240, 240);
   background: hsla(99, 7%, 13%, 0.4);
   cursor: text;
   color: white;
 }
 
-button {
-  padding: 0.4rem;
-  width: 60%;
-  border-radius: 7px;
+.forms label {
+  font-size: 1.4rem;
+}
+
+.forms button {
+  width: 100%;
+  padding: 0.6rem;
+  font-size: 1rem;
+  font-weight: 700;
 }
 
-button:active {
+.forms button:active {
   background-color: transparent;
   color: white;
 }
 
-/* tablet */
-@media screen and (min-width: 706px) {
-  .forms {
-    width: 550px;
-    margin: auto;
-  }
-}
-@media screen and (min-width: 768px) and (orientation: portrait) {
-  .heading-contact h1 {
-    font-size: 4rem;
-    padding: 3rem;
-  }
 
-  .paragraph-contact p {
-    padding: 3rem;
-    font-size: 1.5rem;
-  }
-  .forms form {
-    padding: 2rem;
-  }
-}
+@media only screen and (max-width: 800px) {
```

Comment:
You rarely need more than one media query. You should be able to achieve 90% of the responsiveness with one query. The rest are for aligning things and small details.